tests/gocase/unit/type/hash/hash_test.go

```diff
@@ -783,6 +783,53 @@ func TestHash(t *testing.T) {
 
 			require.Equal(t, []interface{}{"field1", "some-value", "field2", ""}, rdb.Do(ctx, "HrangeByLex", testKey, "[a", "[z").Val())
 		})
+
+		t.Run("HRandField count is positive", func(t *testing.T) {
+			testKey := "test-hash-1"
+			require.NoError(t, rdb.Del(ctx, testKey).Err())
+			require.NoError(t, rdb.HSet(ctx, testKey, "key1", "value1", "key2", "value2", "key3", "value3").Err())
+			result1, err1 := rdb.HRandField(ctx, testKey, 5).Result()
+			require.NoError(t, err1)
+			require.Len(t, result1, 3)
+			require.Equal(t, []string{"key1", "key2", "key3"}, result1)
+			result3, err3 := rdb.HRandField(ctx, testKey, 2).Result()
+			require.NoError(t, err3)
+			require.Len(t, result3, 2)
```

Comment:
would you mind check the value is in `{"key1", "value1", "key2", "value2", "key3", "value3"}`?